spring-integration-amqp/src/main/java/org/springframework/integration/amqp/support/MappingUtils.java

```diff
@@ -32,6 +32,10 @@
  */
 public class MappingUtils {
 
+	private MappingUtils() {
+		super();
```

Comment:
There's a Sonar rule that doesn't like empty blocks; I could have added a comment just to keep sonar happy, but this doesn't hurt - the bytecode is identical:

`INVOKESPECIAL java/lang/Object.<init> ()V`

with or without the `super()`.